client/src/Views/Landing/Landing.js

```diff
@@ -29,6 +30,10 @@ const Landing = props => (
         <Register />
       </Route>
 
+      <Route path="/processGmailToken">
```

Comment:
this route name should be more user friendly - like email-auth-results-dialog

Reply:
Also, it should probably not be in a Switch statement in the future (I know it shouldn't exist here at the moment), because it'll want to be shown on top of whatever page it's currently on as a dialog (Switch only allows one route to be shown at a time)

Reply:
Renamed and removed from switch